docs/src/appendix/forge-library/read_env_var.md

```diff
@@ -0,0 +1,21 @@
+# `read_env_var`
+
+> `fn read_env_var(name: felt252) -> felt252`
```

Comment:
```suggestion
> `fn read_env_var(name: felt252) -> felt252`
```
better to create a module `os` and name it `var` (consistent with rust)

Reply:
What if variable does not exist?

Reply:
By this logic, should current `io` crate be changed to `fs` as well? Because it didn't make that much sense to have a separate `os` create for env vars if all tests<>system operations were already in `io` crate.

Reply:
> What if variable does not exist?

It will cause an error similar to that when parsing fails. I've added an test for this.